index.js

```diff
@@ -561,6 +561,7 @@ class Squeakquel extends Datastore {
             );
 
             findParams.group = config.aggregationField;
+            delete findParams.order;
```

Comment:
hmm, are you assuming whoever uses `aggregationField` will not need any sorting because `aggregationField` is dictated to be used for `SELECT _field,  COUNT(_field)....` type of query?

Reply:
For now that’s the usecase. But when we start refactoring to provide true ORM via sequalize directly will have to modify this. One of the current shortcomings of API data model.